connact-frontend/src/views/Notification.vue

```diff
@@ -1,9 +1,57 @@
 <template>
   <v-container>
-    <v-row class="text-center">
-      <v-col class="mb-4">
-        <h1 class="display-2 font-weight-bold mb-3">Notification Page</h1>
-      </v-col>
-    </v-row>
+    
+    <projectinvite
+    class="project-invite"
+        v-for="(invite, index) in invites"
+        :key="'project: ' + index"
+        :employee="invite.employee.userName"
+        :event="invite.event.eventName"
+        :requestid="invite.id"/>
   </v-container>
-</template>
\ No newline at end of file
+</template>
+<script>
+// @ is an alias to /src
+import projectinvite from '@/components/ProjectInvite.vue'
+
+export default {
+  name: 'test',
+  components: {
+    projectinvite
+  },
+  data() {
+    return {
+      projects: [
```

Comment:
monkaS hardcoded projects, haal deze is snel weg jongeman.

Reply:
no